salt/elife-bot/init.sls

```diff
@@ -55,6 +55,14 @@ elife-bot-settings:
         - require:
             - elife-bot-repo
 
+elife-bot-crossref-cfg:
+    file.managed:
+        - user: {{ pillar.elife.deploy_user.username }}
+        - name: /opt/elife-bot/crossref.cfg
```

Comment:
I assume the library is deployed as a dependency of elife-bot, that's why the configuration is in this folder rather than in its own `/opt/elife-crossref-xml-generation`

Reply:
I think it would be a dependency when activities run, because they would expect a file named ``crossref.cfg`` to be found in the elife-bot root directory when they try to import the elifecrossref module. In order to run test cases while not including a ``crossref.cfg`` in the elife-bot project root directory itself, it seemed better to be able to specify the location of the cfg file used.

Putting the config file into the project using the library seemed to make sense to me. Then we can override any values without having to fiddle with the default config file in ``elife-crossref-xml-generation``, which I'd hoped would be general purposes and used by anyone. Its config file is to satisfy test scenarios, although it currently includes the config values for elife at the time of intended deployment.